src/components/orders/DetailsTable/index.tsx

```diff
@@ -107,6 +110,13 @@ export function DetailsTable(props: Props): JSX.Element | null {
     return null
   }
 
+  const onCopy = (label: string): void =>
+    triggerEvent({
+      category: 'Order details',
+      action: 'Copy',
```

Comment:
@anxolin Might want to be more specific about what is being copied here. E.g. 'Copy tx hash'  or 'Copy account address'.

Reply:
isn't that what `category` is for?

Reply:
I'm being specific. I specify that in the `label` : orderId / tx / address

Reply:
I did a small change so the labels are now: `orderId` / `settlementTx` / `ownerAddress`